lock/client.go

```diff
@@ -14,7 +14,8 @@
 
 package lock
 
-type LockClient interface {
+// Client is a generic interface for a lock
+type Client interface {
```

Comment:
I agree. I removed the hard requirement of passing golint from the test script. Now it just prints out the recommendations, but passes either way.